customScripts/expansionsettings.gd

```diff
@@ -2,12 +2,14 @@
 
 ###---Variables: These can safely be altered---### Still in Progress, will be edited through In-Game Settings UI eventually
 
-var modversion = 1.6
+var modversion = 1.5
```

Comment:
This value is reverted,

Reply:
checked out from main and changed/added only what I wanted for next commit.

Reply:
fixed for next commit